common/scripts/x86_64/CentOS_7/remote/installRedis.sh

```diff
@@ -14,11 +14,12 @@ set_defaults() {
 install_redis() {
   echo "installing redis"
   sudo yum install -y epel-release
-  sudo yum update -y
```

Comment:
This causes packages to "upgrade" (in `apt`'s terms) and is not equivalent to `apt-get update`.